tests/unit/test_ut_scenario_loader.py

```diff
@@ -160,70 +160,94 @@ def test_is_def_filter(self):
         with tempfile.TemporaryDirectory() as dtmp:
             self.create_scenarios(dtmp, levels=3)
             self.create_tests(dtmp, levels=3)
+
+            with open(os.path.join(dtmp, 'test1.yaml'),
+                      'w') as fd:
+                fd.write(FAKE_SCENARIO)
             with mock.patch.object(utils, 'DEFS_TESTS_DIR', dtmp):
-                utils.TemplatedTest('test1.yaml', {}, [], [], [], '/')()(self)
+                utils.TemplatedTest(
+                    os.path.join(dtmp, 'test1.yaml'),
+                    {}, [], [], [], '/')()(self)
 
     def test_scenarios(self):
-        with tempfile.TemporaryDirectory() as dtmp:
+        with tempfile.TemporaryDirectory() as dtmp,\
+             mock.patch.object(utils, 'DEFS_DIR', dtmp),\
+             mock.patch.object(utils, 'DEFS_TESTS_DIR', dtmp + '/tests'):
```

Comment:
Any chance we could use parenthesis instead of slash, it just hurts my eyes a little less:

```
        with (tempfile.TemporaryDirectory() as dtmp,
              mock.patch.object(utils, 'DEFS_DIR', dtmp),
              mock.patch.object(utils, 'DEFS_TESTS_DIR', dtmp + '/tests')):
```

Reply:
Of course, that looks better to my eyes too.

Reply:
@dosaboy it seems like "with" with parantheses introduced in python 3.10 so the py3.8 job fails in CI.